### PR TITLE
Fix create category curl example

### LINKED ISSUE #3963: Trailing coma in Category create example
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

There is a trailing comma in example body.

```
curl --location --request POST 'https://medusa-url.com/admin/product-categories' \
--header 'Authorization: Bearer {api_token}' \
--header 'Content-Type: application/json' \
--data-raw '{
    "name": "Skinny Jeans",
}'
```

```
SyntaxError: Unexpected token } in JSON at position 30
    at JSON.parse (<anonymous>)
```

### How can this issue be resolved?

Remove trailing comma from example fix the error

### Are you interested in working on this issue?

- [X] I would like to fix this issue

### PR DESCRIPTION
**Pull request by LeonAlvarez**
Fix trailing comma on category create example, which causes error:

```
SyntaxError: Unexpected token } in JSON at position 30
    at JSON.parse (<anonymous>)
```

Resolves #3963

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thanks for catching this and opening a fix! However, the files under `docs/api` are automatically generated. Can you instead make the fix in the comments in `packages/medusa/src/api/routes/admin/product-categories/create-product-category.ts`?  Also, if possible, would be great to make the fix in the docs as well in `modules/products/admin/manage-categories` in the Create Category section.

**Comment by LeonAlvarez** [comment]
> Thanks for catching this and opening a fix! However, the files under `docs/api` are automatically generated. Can you instead make the fix in the comments in `packages/medusa/src/api/routes/admin/product-categories/create-product-category.ts`? Also, if possible, would be great to make the fix in the docs as well in `modules/products/admin/manage-categories` in the Create Category section.

Noted, fixed as indicated

File: docs/content/modules/products/admin/manage-categories.mdx [hunk 1]
```diff
@@ -157,7 +157,7 @@ const CreateCategory = () => {
 
   const handleCreate = () => {
     createCategory.mutate({
-      name: "Skinny Jeans",
+      name: "Skinny Jeans"
```
Comment (shahednasser) [inline docs/content/modules/products/admin/manage-categories.mdx]: The fix should be made in the curl example instead (line 198)

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: docs/content/modules/products/admin/manage-categories.mdx (hunk 1 above)
Comment (LeonAlvarez) [reply docs/content/modules/products/admin/manage-categories.mdx]: Ups, fixed.

**Review by shahednasser — APPROVED** [review]
LGTM! Thanks for the help!

**Pull request merged**